Source/santad/DataLayer/SNTRuleTable.h

```diff
@@ -117,4 +117,14 @@
 @property(readonly, nonatomic)
   NSDictionary<NSString *, SNTCachedDecision *> *criticalSystemBinaries;
 
+///
+///  Current supported version of the rule table schema.
+///
+- (uint32_t)currentSupportedVersion;
+
+///
+/// The current version of the rule table schema.
+///
+@property(readonly, nonatomic) uint32_t currentVersion;
```

Comment:
I think this should be a readonly property on SNTDatabaseTable that returns the version from SQLite (`[db userVersion]`) as that will always match what the initializer is provided. This would replace the `currentSchemaVersion` property